Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.py

```diff
@@ -2171,25 +2198,25 @@ def file_sigma_analysis_command(client: Client, args: dict) -> CommandResults:
     """Get last sigma analysis for a given file"""
     file_hash = args['file']
     only_stats = argToBoolean(args['only_stats'])
-    raw_response = client.get_file_sigma_analysis(file_hash)
+    raw_response = client.file(file_hash)
```

Comment:
Are you sure that the new `raw_response` contains all the data as in the old one and that there are no data that might be missed in the new response?
I'm asking because this might break backwards if a user uses the context output given from this command and now this output will be missed.

Reply:
The sigma analysis command hasn't been working for months so there are is no problem with the change. In any case, I have just changed the raw_response including just the important data in a new commit. It contains all the necessary data.

Reply:
Hi @yaakovpraisler, thank you for your comments! :)

Let me know your availability if you want a call for a demo. Thank you very much!